crates/aptos/src/node/mod.rs

```diff
@@ -288,6 +287,16 @@ impl RegisterValidatorCandidate {
     }
 }
 
+#[derive(Parser)]
+/// Register the current account as a Validator candidate
+pub struct RegisterValidatorCandidate {
+    #[clap(flatten)]
+    pub(crate) txn_options: TransactionOptions,
+
+    #[clap(flatten)]
+    pub(crate) validator_config_args: ValidatorConfigArgs,
```

Comment:
where does this config come from? how to update those values?

Reply:
This is in the PR.

I think since the consensus public key is not required, you shouldn't provide all of these args, but instead a subset

Reply:
hmm, but it reads from a file? how do we update that file?

Reply:
look at lines 218-285 above. 
it allows people to set either:

  `Must provide either --validator-config-file or --validator-host`

Reply:
The file come straight out of the Genesis tooling.  You can update the file accordingly, as they're just simply named in yaml